package.json

```diff
@@ -32,6 +32,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-feather": "^2.0.10",
+    "react-icons": "^4.12.0",
```

Comment:
remove this library since it's too big

Reply:
The google icon is not present in "react-feather" , that's why I have used react-icons.
I will remove the "react-icons" .

Reply:
Removed the react-icons